modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -507,7 +509,9 @@ public ObjectField getObjectField(
 	@Override
 	public List<ObjectField> getObjectFields(long objectDefinitionId) {
 		List<ObjectField> objectFields =
-			objectFieldPersistence.findByObjectDefinitionId(objectDefinitionId);
+			objectFieldPersistence.findByObjectDefinitionId(
+				objectDefinitionId, QueryUtil.ALL_POS, QueryUtil.ALL_POS,
+				new ObjectFieldCreateDateComparator(true));
```

Comment:
@ricardocousolr can you create a new service method (like this [one](https://github.com/liferay/liferay-portal/blob/43a8fc59fac7ddba761602493e22d04eec7b94d4/modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java#L222)) and change the call only where it's necessary?

Reply:
@carolmariaabb I've implemented your correction in 3 extra commits. I used the new method in `com.liferay.object.admin.rest.internal.resource.v1_0.ObjectDefinitionResourceImpl` because I don't think it can be moved all the way up to `com.liferay.object.web.internal.object.definitions.portlet.action.ExportObjectDefinitionMVCResourceCommand`, but correct me if I'm wrong.